content/en/docs/release-oversight/test-case-analyzer.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "Test Case Analyzer"
+description: Introduction of a new tool to analyze test cases across multiple job runs.
+---
+
+### Summary
+On top of the existing aggregator framework, a new tool (test case analyzer) is introduced to analyze test results from a much broader candidate job runs. 
+
+### Further Improvement to CI Payload Signals
+CI and nightly payloads run many different jobs. Some of those jobs are blocking while the majority are informing. Failure of any blocking jobs will reject the payload. But informing job results are not utilized in any automated way to provide valuable CI signals. 
+
+With the introduction of the current aggregator framework, many job runs can be started for the same job and the test results from those job runs are aggregated to provide much better CI signals. This aggregated result can be used to form a blocking signal to the overall payload. This has been working really well to catch issues that were not possible by just analyzing single job runs in isolation. 
```

Comment:
This is accurate but I suspect you could drop this paragraph and be no worse off. Up to you just a suggestion for thought.